versioned_docs/version-V3/guides/governance/license-modifications.md

```diff
@@ -0,0 +1,171 @@
+
+- `node`: `namehash('v3-core-license-date.uniswap.eth')` (`0x0505ec7822d61b4cfb294f137d1a7f0ceedf162f555a4bf2f4be58a07cf266c5`) or `namehash('v3-core-license-grants.uniswap.eth')` (`0xa35d592ec6e5289a387cba1d5f82be794f495bd5a361a1fb314687c6aefea1f4`)
+- `key`: A suitable label, such as `notice`.
+- `value`: The text of the change. Note that text may already be associated with the subdomain in question. If it does, it can be reviewed at the following URLs for either [v3-core-license-date](https://app.ens.domains/name/v3-core-license-date.uniswap.eth/details) or [v3-core-license-grants](https://app.ens.domains/name/v3-core-license-grants.uniswap.eth/details), and appended to as desired.
+
+Note: [`setContentHash`](https://docs.ens.domains/contract-api-reference/publicresolver#set-content-hash) may also be used to associate text with a subdomain, but `setText` is presented above for simplicity.
+
+These contract function calls should then be encoded into a governance proposal, and approved by Uniswap Governance.
+
+</details>
+
+## Proposals
+
+Proposals are submitted to Uniswap Governance via the [Governance Bravo Delegate contract](https://etherscan.io/address/0x53a328f4086d7c0f1fa19e594c9b842125263026#code). NPM packages for consuming the governance contract ABIs, and details on previous versions, are available [here](https://docs.uniswap.org/protocol/concepts/governance/overview).
+
+<details>
+    <summary> Governor Bravo #propose Parameters </summary>
+
+Deployment reference available [here.](https://etherscan.io/address/0x53a328f4086d7c0f1fa19e594c9b842125263026#code)
+
+```solidity
+/**
+    * @notice Function used to propose a new proposal. Sender must have delegates above the proposal threshold
+    * @param targets Target addresses for proposal calls
+    * @param values Eth values for proposal calls
+    * @param signatures Function signatures for proposal calls
+    * @param calldatas Calldatas for proposal calls
+    * @param description String description of the proposal
+    * @return Proposal id of new proposal
+    */
+function propose(
+    address[] memory targets,
+    uint[] memory values,
+    string[] memory signatures,
+    bytes[] memory calldatas,
+    string memory description
+) public returns (uint)
+
+```
+
+</details>
+
+## Populating Proposal Calldata
+
+Below is an example of using a scripting environment to generate a proposal. This is for educational purposes only - that example assumes access to a private key with a sufficient amount of delegated UNI to submit a proposal, which may be an insecure practice. There are several ways to generate a proposal transaction and submit it to Ethereum; this example should only be used for reference and not in production.
+
+<details>
+<summary> Populating `Propose` Calldata </summary>
+
+```typescript
+import { ethers } from "hardhat";
+import { waffle } from "hardhat";
```

Comment:
all imports from the same package should be grouped - probably best to have a code env with prettier and eslint set up that you generate these code samples in to catch stuff like this